### ISSUE TITLE
It possible run in snap?

### ISSUE THREAD

**Issue by jershell** [waydroid/waydroid#115]
Yesterday I tried to install waydroid on my ubuntu 20.04. After the update, I had crash gdm3 by mutter-common 3.38 . Ubuntu 20.04 LTS use mutter 3.36. It led to the idea run waydroid inside snap. It's possible?

**Comment by PhilDevProg** [comment]
It should be possible.
But Flatpak would be a better option because it's faster and more popular than snap.
There is already a discussion about Flatpak: #64

**Comment by enz0nun3s15** [comment]
One of the largest problem right now with waydroid/anbox is installing, I guess if there was a Flatpack/AppImage you would still need to install Wayland and the kernel modules yourself. With snap you could truly pack everything and make install it in a single command.

**Comment by rokejulianlockhart** [comment]
Yeah, whereas `flatpak` is not immediately feasible, `snap` should be, due to `--classic`.

**Comment by Twig6943** [comment]
Any updates on this? Waydroid is the only app I use daily that doesn't have a universal pkg https://github.com/Twig6943/dotfiles/blob/main/rebos/EndeavourOS/gen.toml

**Comment by YamiYukiSenpai** [comment]
Is it possible, at least, to use Snap's LXD/LXC to run Waydroid?
